packages/frontend/src/components/SettingsValidator/index.tsx

```diff
@@ -65,10 +65,29 @@ const SettingsValidator: FC<{ projectUuid: string }> = ({ projectUuid }) => {
         return <IconBox icon={IconTable} color="indigo.6" />;
     };
 
+    const [maxTableHeight, setMaxTableHeight] = useState(0);
```

Comment:
What is the advantage of calculating the max height based on the first row height instead of a hardcoded max height? eg: 500px

Reply:
Dynamic height for a specific number of rows I guess? 😅  That way we can change the min of 6 to whatever number and it will just work. 
Heights of rows can differ between devices so calculating that is more adaptable IMO

Reply:
now with the `500px` you've suggested, thanks @ZeRego !

![image](https://github.com/lightdash/lightdash/assets/7611706/bc360f59-69b4-4495-a6df-6cbffdc3d79b)